misc/yosys-config.in

```diff
@@ -37,11 +37,11 @@ help() {
 		echo "  $0 --datdir/simlib.v"
 		echo ""
 	} >&2
```

Comment:
This is still printing to stderr even when exiting normally, is it possible to print to stdout instead when it's called with `--help`?

Reply:
It is doable, should be fine now with latest change.